certbot-apache/certbot_apache/tests/augeas_configurator_test.py

```diff
@@ -63,23 +63,9 @@ def test_finalize_save(self):
 
         self.assertTrue(mock_finalize.is_called)
 
-    def test_recovery_routine(self):
```

Comment:
If we add an implementation of `recovery_routine()` to the configurator, I think we should add at least this test back if not both.